torch/distributed/_tensor/examples/convnext_example.py

```diff
@@ -0,0 +1,256 @@
+import os
+import time
+import warnings
+
+import torch
+import torch.distributed as dist
+import torch.multiprocessing as mp
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.distributed._tensor import (
+    DeviceMesh,
+    distribute_module,
+    distribute_tensor,
+    Replicate,
+    Shard,
+)
+
+warnings.filterwarnings("ignore", category=UserWarning)
+
+WORLD_SIZE = 2
+ITER_TIME = 20
+
+
+class LayerNorm(nn.Module):
+    def __init__(self, normalized_shape, eps=1e-6, data_format=torch.channels_last):
+        super().__init__()
+        self.weight = nn.Parameter(torch.ones(normalized_shape))
+        self.bias = nn.Parameter(torch.zeros(normalized_shape))
+        self.eps = eps
+        self.data_format = data_format
+        if self.data_format not in [torch.contiguous_format, torch.channels_last]:
+            raise NotImplementedError
+        self.normalized_shape = (normalized_shape,)
+
+    def forward(self, x):
+        if self.data_format == torch.channels_last:
+            return F.layer_norm(
+                x, self.normalized_shape, self.weight, self.bias, self.eps
+            )
```

Comment:
note: I'm writing DTensor layer_norm_backward op but so far we only have the forward one.

Reply:
I see. `F.layer_norm` and `nn.LayerNorm` are not actually used in this example.